sdk/eventhub/azure-eventhubs/HISTORY.md

```diff
@@ -1,9 +1,20 @@
 # Release History
 
-## 2019-11-04 5.0.0b6
+## 2019-12-03 5.0.0b6
 
 **Breaking changes**
 
+- `EventData`
+    - Renamed property `application_properties` to `properties`.
+- `EventHubConsumerClient` and `EventHubProducerClient`
+    - Renamed method `get_properties` to `get_eventhub_properties`.
+    - Renamed parameters in constructor: `host` to `fully_qualified_namespace`, `event_hub_path` to `eventhub_name`.
+    - Renamed parameters in `get_partition_properties`: `partition` to `partition_id`.
+- Moved `consumer_group` parameter from `receive` method to the constructor of `EventHubConsumerClient`.
```

Comment:
Especially since consumer_group isn't actually used until receive, is there a reason we shouldn't support it as an optional override param on receive?  This would prevent it from being as aggressive a breaking change, and would still allow scenarios where a user can receive across multiple consumer groups without newing up multiple receiver entities.

Reply:
My understanding is that we are moving to a conceptual model where the ConsumerClient represents receiving from a specific consumer group. Therefore allowing an on-receive override would be counter to that model.

Reply:
I think the reason I'm making a face is that from the perspective of the underlying, this is an artificial constraint, and it limits a more terse syntax of multi-group-consumption.

Obviously I'm coming in late to this discussion so I won't buck it too hard now, but I'm not sold that the "purity" of the model here would be sufficiently disrupted to counter that advantage.  

Anyway; I won't belabor it further, it's not the end of the world, and it's a point I can argue iteratively, especially if anyone complains re: the breaking change.

Reply:
Yeah we can probably punt this one - as it can be added as a non-breaking change if it makes for an elegant customer solution. Though as it currently stands I don't think we have a compelling reason to be different - though I say this only because we really don't have many customer scenarios informing our decisions rather than service-team directives.....